src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.props

```diff
@@ -4,5 +4,6 @@
 		<_JavaInteropReferences>Java.Interop;System.Runtime</_JavaInteropReferences>
 		<DependsOnSystemRuntime Condition=" '$(DependsOnSystemRuntime)' == '' ">true</DependsOnSystemRuntime>
 		<CopyNuGetImplementations Condition=" '$(CopyNuGetImplementations)' == ''">true</CopyNuGetImplementations>
+		<_YieldDuringExecution Condition="'$(_YieldDuringExecution)' == ''">true</_YieldDuringExecution>
```

Comment:
I'd call it something more explicit and generic, like `YieldDuringAsyncTask`, since it's quite obvious to me that this task belongs in MSBuild itself even.

/cc @rainersigwald